main/tests.py

```diff
@@ -1 +1,62 @@
+import pytest
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+
+
+@pytest.mark.django_db
+class TestLoginView:
+
+    def test_enter_login_page(self, client):
+        response = client.get('/login/')
+        assert response.status_code == 200
+
+    def test_successful_login_as_dogowner_user(self, client):
+        dog_owner_user = DogOwner.create('dogowner@address.com', 'dogOwnerUser01', 'password123', 'dog name', 'test',
+                                         'user', 1234567890, 'dog race', 'https://www.google.com/user1', 4, 2, 'M')
+
+        client.force_login(dog_owner_user.user)
+        response = client.get('/login/')
+        assert response.status_code == 200
+        logged_user = response.wsgi_request.user
+        assert logged_user == dog_owner_user.user
+
+    def test_successful_login_as_daycare_user(self, client):
+        day_care_user = DayCare.create('daycare@address.com', 'dayCareUser', 'password123', 'daycare',
+                                       'first description', 10,
+                                       100, 'north', 'tel-aviv', 'street 01')
+
+        client.force_login(day_care_user.user)
+        response = client.get('/login/')
+        assert response.status_code == 200
+        logged_user = response.wsgi_request.user
+        assert logged_user == day_care_user.user
+
+    def test_valid_login_dog_owner_user_info(self, client):
+        dog_owner_user = DogOwner.create('dogowner@address.com', 'dogOwnerUser02', 'password123', 'dog name', 'test',
+                                         'user', 1234567890, 'dog race', 'https://www.google.com/user1', 4, 2, 'M')
+
+        form = {'username': 'dogOwnerUser02',
+                'password': 'password123',
+                }
+        response = client.post('/login/', form, follow=True)
+        assert response.status_code == 200
+        assert dog_owner_user.user == response.wsgi_request.user
```

Comment:
also save a logged_used before force_login and test that they are not the same.

Reply:
Fixed.